### PR TITLE
Feature: Field Deriver for PublicKey

### PR DESCRIPTION
**Pull request by MartinMinkov**

### CODE REVIEW

**Comment by mitschabaude** [comment]
LGTM!

**Review by mitschabaude — APPROVED** [review]

File: src/lib/fields_derivers_snapps/fields_derivers_snapps.ml [hunk 1]
```diff
@@ -96,6 +96,13 @@ module Derivers = struct
     iso_string obj ~name:"Field" ~doc:"String representing an Fp Field element"
       ~to_string:Field.to_string ~of_string:Field.of_string
 
+  let public_key obj : _ Unified_input.t =
+    let module Public_key = Signature_lib.Public_key in
```
Comment (MartinMinkov) [inline src/lib/fields_derivers_snapps/fields_derivers_snapps.ml:100]: TODO: Fix this

File: src/lib/fields_derivers_snapps/fields_derivers_snapps.ml [hunk 2]
```diff
@@ -96,6 +96,12 @@ module Derivers = struct
     iso_string obj ~name:"Field" ~doc:"String representing an Fp Field element"
       ~to_string:Field.to_string ~of_string:Field.of_string
 
+  let public_key obj : _ Unified_input.t =
+    iso_string obj ~name:"Field"
```
Comment (bkase) [inline src/lib/fields_derivers_snapps/fields_derivers_snapps.ml:100]: Can you switch this to `PublicKey`?

**Review by bkase — COMMENTED** [review]
just one little nit (feel free to just commit this to `feature/fields-derivers` directly)

**Pull request merged**